lms/djangoapps/teams/static/teams/js/views/team_profile.js

```diff
@@ -41,7 +41,8 @@
                         discussionTopicID = this.model.get('discussion_topic_id'),
                         isMember = TeamUtils.isUserMemberOfTeam(memberships, this.context.userInfo.username),
                         isAdminOrStaff = this.context.userInfo.privileged || this.context.userInfo.staff,
-                        isInstructorManagedTopic = TeamUtils.isInstructorManagedTopic(this.topic.attributes.type);
+                        isInstructorManagedTopic = TeamUtils.isInstructorManagedTopic(this.topic.attributes.type),
+                        maxTeamSize = this.topic.getMaxTeamSize(this.context.courseMaxTeamSize);
```

Comment:
Nit: call this variable `courseMaxTeamSize` for consistency.

Reply:
well, this is now the actual resolved max team size (which can be the courseMaxTeamSize but isn't necessarily)

Reply:
ah, I missed that this as a call to `getMaxTeamSize()`